src/java.base/share/classes/java/util/Locale.java

```diff
@@ -1645,18 +1645,19 @@ public String toLanguageTag() {
      * </pre></ul>
      *
      * <p>This implements the 'Language-Tag' production of BCP47, and
-     * so supports grandfathered (regular and irregular) as well as
+     * so supports legacy (regular and irregular, referred to as
+     * {@code Type: grandfathered} in BCP47) as well as
```

Comment:
You might consider putting quotes around "Type; grandfathered" here, otherwise looks good.

Reply:
Yes, having that in quotes instead of in {@code} would be consistent with the rest of Locale.java.  I will change that.